root/scripts/authUtils.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable import/prefer-default-export */
+const FIREBASE_SIGNUP_URL = 'https://www.googleapis.com/identitytoolkit/v3/relyingparty/signupNewUser';
+const FIREBASE_LOGIN_URL = 'https://www.googleapis.com/identitytoolkit/v3/relyingparty/verifyPassword';
+const FIREBASE_KEY = 'AIzaSyCSXC0gS7gdHANxVdisyYg1eRg18znec_k';
```

Comment:
we should eventually move these into GitHub secrets 😄

Reply:
Yeah for sure